opam.export

```diff
@@ -31,7 +31,6 @@ roots: [
   "ppx_bitstring.4.1.0"
   "ppx_deriving.5.2.1"
   "ppx_deriving_yojson.3.6.1"
-  "ppx_optcomp.v0.14.3"
```

Comment:
This is only removed from roots instead of installed. It's used as a transitive deps for another lib for now. We'll get rid of it when bumping to corev0.16